### PR TITLE
Added comment about linting

### PR DESCRIPTION
**Pull request by laserlab**
# Added a comment about linting in the PR template

## Contents
  - added a comment that the linting result should be checked
  - pull request template

## Reviewer
  - @SchrodingersStruggle

### CODE REVIEW

File: docs/pull_request_template.md [hunk 1]
```diff
@@ -3,6 +3,9 @@ For this assignment all pull requests going  forward should contain the followin
 # Title
   - A ledgible title that briefly describes the contents of the pull request.
 
+## linting
+  - does you code lint?
```
Comment (SchrodingersStruggle) [inline docs/pull_request_template.md:7]: Is this supposed to say:
```suggestion
  - Does your code lint?
```

**Review by SchrodingersStruggle — COMMENTED** [review]
Minor spelling/grammar to check in my comment.

File: docs/pull_request_template.md (hunk 1 above)
Comment (SchrodingersStruggle) [reply docs/pull_request_template.md:7]: Also, is Linting meant to have a capital L in the beginning?

Comment (laserlab) [reply docs/pull_request_template.md]: fixed

**Comment by laserlab** [comment]
@SchrodingersStruggle jst fixed the typos. Ready to merge

**Review by SchrodingersStruggle — APPROVED** [review]
Looks good now

**Pull request merged**